frontend/src/server/db/index.ts

```diff
@@ -0,0 +1,18 @@
+import { drizzle } from "drizzle-orm/postgres-js";
```

Comment:
I don't believe this file should be needed anymore, can probably delete (correct me if I'm wrong)